v1_0/assets/tasks.pbtxt

```diff
@@ -0,0 +1,205 @@
+    type: IMAGENET
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/imagenet_tiny.zip"
+    groundtruth_src: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/assets/imagenet_val_tiny.txt"
+  }
+  model {
+    id: "IC_tpu_uint8"
+    name: "MobileNetEdgeTPU 224 1.0 Uint8"
+    offset: 1
+    tags: "uint8"
+    score_max: 1000
+  }
+  model {
+    id: "IC_tpu_float32"
+    name: "MobileNetEdgeTPU 224 1.0 F32"
+    offset: 1
+    tags: "float32"
+    score_max: 1500
+  }
+  min_query_count: 1024
+  min_duration_ms: 60000
+}
+
+task {
+  name: "Object detection"
+  dataset {
+    name: "COCO 2017 validation set"
+    type: COCO
+    path: "/sdcard/mlperf_datasets/coco/img"
+    groundtruth_src: "/sdcard/mlperf_datasets/coco/coco_val_full.pbtxt"
+  }
+  lite_dataset {
+    name: "Open images subset for detection"
+    type: COCO
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/coco.zip"
+    groundtruth_src: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/assets/coco_val.pbtxt"
+  }
+  test_dataset {
+    name: "Coco dataset for instrument test"
+    type: COCO
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/coco_tiny.zip"
+    groundtruth_src: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/assets/coco_val_tiny.pbtxt"
+  }
+  model {
+    id: "OD_uint8"
+    name: "SSD-MobileDET UInt8"
+    offset: 1
+    tags: "uint8"
+    score_max: 2000
+  }
+  model {
+    id: "OD_float32"
+    name: "SSD-MobileDET F32"
+    offset: 1
+    tags: "float32"
+    score_max: 5000
+  }
+  min_query_count: 1024
+  min_duration_ms: 60000
+}
+
+task {
+  name: "Image Segmentation"
+  dataset {
+    name: "ADE20K validation set"
+    type: ADE20K
+    path: "/sdcard/mlperf_datasets/ade20k/images"
+    groundtruth_src: "/sdcard/mlperf_datasets/ade20k/annotations"
+  }
+  lite_dataset {
+    name: "Open images subset for segmentation"
+    type: ADE20K
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/ade20k.zip"
+    groundtruth_src: ""
+  }
+  test_dataset {
+    name: "Open images subset for segmentation"
+    type: ADE20K
+    path: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/datasets/ade20k_tiny.zip"
+    groundtruth_src: ""
+  }
+  model {
+    id: "IS_float32"
+    name: "Deeplab-v3 Mobilenet-v2 F32"
+    tags: "float32"
+    score_max: 1800
+  }
+  model {
+    id: "IS_uint8"
+    name: "Deeplab-v3 Mobilenet-v2 UINT8"
+    tags: "uint8"
+    score_max: 2000
+  }
+  model {
+    id: "IS_int8"
+    name: "Deeplab-v3 Mobilenet-v2 INT8"
+    tags: "int8"
+    score_max: 3000
+  }
+  min_query_count: 1024
+  min_duration_ms: 60000
+}
+
+task {
+  name: " Language Understanding"
+  dataset {
+    name: "Squad V1.1 validation set"
+    type: SQUAD
+    path: "@assets/squad_eval_mini.tfrecord"
+    groundtruth_src: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/assets/squad_groundtruth.tfrecord"
+  }
```

Comment:
Here and below:
What the reason to use an "external" groundtruth file for an "internal" dataset?
Why not move the dataset to the repository too?